cross/curaengine/digests

```diff
@@ -0,0 +1,3 @@
+15.04.tar.gz SHA1 1a1d2c8f1b3311fbeb5b87fe32b8051508263abe
```

Comment:
I believe setting `PKG_DIST_FILE` in `cross/curaengine/Makefile`, allows for the downloaded file to look like `$(PKG_NAME)-$(PKG_VERS).$(PKG_EXT). Makes it easier to locate in the distrib dir.